static/css/header.css

```
@@ -334,4 +334,70 @@
+}
+
+.self-promo-inner {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    width: -webkit-fill-available;
+}
+
+.self-promo-inner a {
+    text-align: center;
```

Stylelint (17.7.0)

[error] Duplicate property "text-align" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)